static/js/unread_ui.js

```diff
@@ -84,7 +84,7 @@ export function update_unread_counts() {
     // Set the unread counts that we show in the buttons that
     // toggle open the sidebar menus when we have a thin window.
     set_count_toggle_button($("#streamlist-toggle-unreadcount"), res.home_unread_messages);
-    set_count_toggle_button($("#userlist-toggle-unreadcount"), res.private_message_count);
+    set_count_toggle_button($("#userlist-toggle-unreadcount"), res.private_non_bot_message_count);
```

Comment:
Let's add a comment noting that because bots don't appear in the right sidebar, ....

Reply:
Done, this should be ready for another review @timabbott